SPECS/maven-resources-plugin/maven-resources-plugin.spec

```diff
@@ -0,0 +1,206 @@
+
+Summary:        Maven Resources Plugin
+Name:           maven-resources-plugin
+Version:        3.2.0
```

Comment:
why not using the latest (3.3.1) ?

Reply:
bootstrap supports 3.2.0.
The build fails with newer versions.